package.json

```diff
@@ -59,5 +59,5 @@
     "eslint-plugin-ui-testing": "^2.0.1",
     "serve": "^14.2.1"
   },
-  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
-}
\ No newline at end of file
+}
+  "packageManager": "yarn@4.4.1"
```

Comment:
oops. bad merge.

Reply:
ah. I must have fixed it later.

Reply:
this build was still failing 🤔 

I managed to resolve it in #105! 🥳